### PR TITLE
Sepolia Geth node deployment

### PR DESCRIPTION
**Pull request by Marudny**
infra-nimbus: added Sepolia nodes, unified kiln and sepolia ansible configuration.

### CODE REVIEW

File: kiln.tf [hunk 1]
```diff
@@ -4,7 +4,7 @@ module "nimbus_nodes_kiln_hetzner" {
   name   = "metal"
   env    = "nimbus"
   stage  = "kiln"
-  group  = "nimbus-kiln-metal"
+  groups  = [ "nimbus-kiln-metal", "nimbus-sepolia-metal" ]
```
Comment (jakubgs) [inline kiln.tf:7]: We don't need this anymore.

File: ansible/sepolia.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      changed_when: false
+
+- name: Deploy Spolia Testnet nodes
+  become: true
+  serial: '{{ serial|default(1) }}'
+  hosts:
+    - nimbus-sepolia-metal
+  tasks:
+    - include_vars:
+        file: group_vars/nimbus-sepolia-metal.yml
```
Comment (jakubgs) [inline ansible/sepolia.yml:19]: Wouldn't it be cleaner to move those to `vars` to make it clear those are loaded differently?

File: ansible/inventory/test [hunk 3]
```diff
@@ -143,6 +143,9 @@ windows-01.he-eu-hel1.nimbus.prater
 [nimbus-ropsten-metal]
 metal-01.he-eu-hel1.nimbus.ropsten
 
+[nimbus-sepolia-metal]
+metal-01.he-eu-hel1.nimbus.kiln
```
Comment (jakubgs) [inline ansible/inventory/test:147]: Also no longer necessary.

**Review by jakubgs — APPROVED** [review]
Looking good. But probably needs some comments in the vars files or/and the playbook explaining why we couldn't just use the `group_vars` files.

File: kiln.tf (hunk 1 above)
Comment (Marudny) [reply kiln.tf:7]: Have removed.

File: ansible/inventory/test (hunk 3 above)
Comment (Marudny) [reply ansible/inventory/test:147]: Have removed.

File: ansible/sepolia.yml (hunk 2 above)
Comment (Marudny) [reply ansible/sepolia.yml:19]: Indeed, have created `vars` directory and moved `sepolia` and `kiln` files there.

**Review by jakubgs — APPROVED** [review]
lgtm

**Pull request merged**